Automattic/wp-calypso
# Add support for managing WordAds settings of Atomic sites

#### Changes proposed in this Pull Request

This PR adds support for managing the WordAds settings of Atomic sites directly in Calypso, rather than redirecting users to Jetpack > Settings > Traffic.

<img width="400" alt="Screen Shot 2021-06-16 at 12 03 36" src="https://user-images.githubusercontent.com/1233880/122200038-f4116c00-ce9a-11eb-8431-b1a123ecdab2.png">

This helps with the effort of removing the "Jetpack > Settings" submenu needed for #51342, since right now the WordAds settings of Atomic sites can be managed only from that submenu.

#### Testing instructions

- Pick an Atomic site and go to Jetpack > Settings > Traffic.
- Customize the Ads settings as you wish.
- Go to Tools > Earn > View ad dashboard > Settings.
- Make sure the Ads settings values match with Jetpack > Settings > Traffic.
- Customize the Ads settings as you wish.
- Go to Jetpack > Settings > Traffic.
- Make sure the Ads settings values match with Tools > Earn > View ad dashboard > Settings.
- Make sure there are no regressions on Simple sites.
